packages/components/src/components/Button/styles/buttonStyles.ts

```diff
@@ -3,7 +3,7 @@
 import { sv } from '#/utilities/styleUtilities';
 
 export const buttonStyles = sv({
-    base: 'tw-group tw-border tw-box-box tw-relative tw-flex tw-flex-row tw-gap-2 tw-items-center tw-justify-center tw-cursor-pointer tw-outline-none tw-font-body tw-font-medium',
+    base: 'tw-group tw-border tw-box-box tw-relative tw-flex tw-flex-row tw-gap-2 tw-items-center tw-justify-center tw-cursor-pointer tw-font-body tw-font-medium',
```

Comment:
Out of scope: I think `tw-box-box` doesn't exist

Reply:
removed